docs/src/modules/components/MarkdownElement.js

```diff
@@ -15,40 +15,38 @@ const Root = styled('div')(({ theme }) => ({
     margin: theme.spacing(2, 'auto'),
     padding: theme.spacing(2),
     backgroundColor: blueDark[800],
+    color: '#f8f8f2', // fallback color until Prism's theme is loaded
     colorScheme: 'dark',
     direction: 'ltr',
     borderRadius: theme.shape.borderRadius,
     border: '1px solid',
     borderColor: blueDark[700],
     overflow: 'auto',
     WebkitOverflowScrolling: 'touch',
+    fontSize: theme.typography.pxToRem(13),
     maxWidth: 'calc(100vw - 32px)',
     maxHeight: '400px',
     [theme.breakpoints.up('md')]: {
       maxWidth: 'calc(100vw - 32px - 16px)',
     },
   },
-  '& code, & code[class*="language-"]': {
-    direction: 'ltr',
-    display: 'inline-block',
+  '& code': {
     ...theme.typography.body2,
-    fontSize: theme.typography.pxToRem(13),
-    fontFamily: theme.typography.fontFamilyCode,
+    fontFamily: brandingDarkTheme.typography.fontFamilyCode,
     fontWeight: 400,
     WebkitFontSmoothing: 'subpixel-antialiased',
-    padding: '0 5px',
-    borderRadius: 5,
+    // Reset for Safari
+    // https://github.com/necolas/normalize.css/blob/master/normalize.css#L102
+    fontSize: '1em',
```

Comment:
The fix for Safari font size is to not apply the custom font size on the `code` but on the `pre` element.